src/components/views/dialogs/DeactivateAccountDialog.tsx

```diff
@@ -180,7 +184,7 @@ export default class DeactivateAccountDialog extends React.Component<IProps, ISt
                     <InteractiveAuth
                         matrixClient={MatrixClientPeg.get()}
                         authData={this.state.authData}
-                        makeRequest={this.onUIAuthComplete}
+                        makeRequest={this.onUIAuthComplete as any}
```

Comment:
Comment or something to denote why we're using `any`, with possible means to correct our bad behaviour.

tldr: `// TODO: Types`